tests/TransactionBuilderTest.php

```diff
@@ -0,0 +1,220 @@
+<?php
+declare(strict_types=1);
```

Comment:
If we're using strict types here, would it also make sense to do so in the new `TransactionManager.php` file being introduced?

Reply:
Done